specification/managedccf/data-plane/Microsoft.mCCF/preview/2023-03-03-preview/openapi.json

```diff
@@ -0,0 +1,1912 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CCF governance",
+    "version": "2023-02-20",
+    "x-cadl-generated": [
+      {
+        "emitter": "@azure-tools/cadl-autorest"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "tags": [],
+  "paths": {
+    "/gov/ack": {
+      "post": {
+        "operationId": "Ack",
+        "summary": "Provide a member endorsement of a service state digest",
+        "parameters": [
+          {
+            "name": "state_digest",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/StateDigest"
+            }
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "There is no content to send for this request, but the headers may be useful. "
+          }
+        }
+      }
+    },
+    "/gov/ack/update_state_digest": {
+      "post": {
+        "operationId": "UpdateStateDigest",
+        "summary": "Update and fetch a service state digest, for the purpose of member endorsement",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/StateDigest"
+            }
+          }
+        }
+      }
+    },
+    "/gov/commit": {
+      "get": {
+        "operationId": "Commit",
+        "summary": "Current commit level",
+        "description": "Latest transaction ID that has been committed on the service",
+        "parameters": [
+          {
```

Comment:
Not sure what these parameters are for, would be good to add a `description` to each

Reply:
Will add descriptions, they are ways to query the view history for a client, which is useful when trying to compute efficiently client-side if a large number of transactions has been committed or not.